server/routes/timetable.test.ts

```diff
@@ -3,11 +3,24 @@ import request from 'supertest'
 import * as cheerio from 'cheerio'
 import { appWithAllRoutes } from './testutils/appSetup'
 import config from '../config'
+import VisitSessionsService from '../services/visitSessionsService'
+
+jest.mock('../services/visitSessionsService')
 
 let app: Express
+const systemToken = async (user: string): Promise<string> => `${user}-token-1`
+
+const visitSessionsService = new VisitSessionsService(
+  null,
+  null,
+  null,
+  systemToken,
+) as jest.Mocked<VisitSessionsService>
 
 beforeEach(() => {
-  app = appWithAllRoutes({})
+  visitSessionsService.getSessionSchedule.mockResolvedValue([])
```

Comment:
Tests were failing because this service needed mocking. Just returning empty schedule for now.